### PR TITLE
Add Shell action mapper

### LINKED ISSUE #24: Implement Shell mapper

### PR DESCRIPTION
**Pull request by sprzedwojski**
Closes #24

### CODE REVIEW

File: oozie-to-airflow/mappers/shell_mapper.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        self.resource_manager = el_utils.replace_el_with_var(res_man_text, params=self.params, quote=False)
+        self.name_node = el_utils.replace_el_with_var(name_node_text, params=self.params, quote=False)
+        self._parse_config()
+        self._parse_params()
+        cmd_node = self.oozie_node.find("exec")
+        arg_nodes = self.oozie_node.findall("argument")
+        cmd = " ".join([cmd_node.text] + [x.text for x in arg_nodes])
+        self.bash_command = el_utils.convert_el_to_jinja(cmd, quote=False)
+
+    def _parse_params(self):
```
Comment (potiuk) [inline oozie-to-airflow/mappers/shell_mapper.py]: What are those params?. I could not find it in https://oozie.apache.org/docs/5.0.0/DG_ShellActionExtension.html

File: oozie-to-airflow/examples/shell/workflow.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <exec>java</exec>
+            <argument>-version</argument>
+            <!--<file>${EXEC}</file>--> <!-- TODO Add the binary upload when file mapper ready -->
+            <capture-output/> <!-- This can be ignored by the converter -->
+        </shell>
+<!--        <ok to="check-output"/>-->
+        <ok to="end"/>
+        <error to="fail"/>
+    </action>
+<!--    TODO uncomment once decision has been implemented -->
```
Comment (potiuk) [inline oozie-to-airflow/examples/shell/workflow.xml]: Remove it :) ?

File: oozie-to-airflow/examples/shell/workflow.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+<!--            <case to="end">-->
+<!--                ${wf:actionData('shell-node')['my_output'] eq 'Hello Oozie'}-->
+<!--            </case>-->
+<!--            <default to="fail-output"/>-->
+<!--        </switch>-->
+<!--    </decision>-->
+    <kill name="fail">
+        <message>Shell action failed, error message[${wf:errorMessage(wf:lastErrorNode())}]</message>
+    </kill>
+    <!--<kill name="fail-output">-->
```
Comment (potiuk) [inline oozie-to-airflow/examples/shell/workflow.xml]: Same here :)

File: oozie-to-airflow/mappers/shell_mapper.py (hunk 1 above)
Comment (sprzedwojski) [reply oozie-to-airflow/mappers/shell_mapper.py]: You're right, it shouldn't be here. Copy-pasting gone wrong ;) I'll remove it.

File: oozie-to-airflow/examples/shell/workflow.xml (hunk 2 above)
Comment (sprzedwojski) [reply oozie-to-airflow/examples/shell/workflow.xml]: Yeah, I'll move it to a Github issue as a TODO.

File: oozie-to-airflow/examples/shell/workflow.xml (hunk 3 above)
Comment (sprzedwojski) [reply oozie-to-airflow/examples/shell/workflow.xml]: Same as above.

File: oozie-to-airflow/mappers/shell_mapper.py (hunk 1 above)
Comment (sprzedwojski) [reply oozie-to-airflow/mappers/shell_mapper.py]: Done

File: oozie-to-airflow/examples/shell/workflow.xml (hunk 2 above)
Comment (sprzedwojski) [reply oozie-to-airflow/examples/shell/workflow.xml]: Done

File: oozie-to-airflow/examples/shell/workflow.xml (hunk 3 above)
Comment (sprzedwojski) [reply oozie-to-airflow/examples/shell/workflow.xml]: Done

**Pull request merged**